package.json

```diff
@@ -56,6 +56,10 @@
     "redux-saga": "1.1.3"
   },
   "dependencies": {
+    "@codemirror/lang-html": "^6.0.0",
+    "codemirror": "^6.0.0",
```

Comment:
note: "codemirror" isn't installing the extensions listed in these other packages